packages/auth-construct/src/construct.ts

```diff
@@ -770,9 +772,7 @@ export class AmplifyAuth
       scopes: external.scopes
         ? this.getOAuthScopes(external.scopes)
         : DEFAULT_OAUTH_SCOPES,
-      flows: {
-        authorizationCodeGrant: true,
-      },
+      flows: DEFAULTS.OAUTH_FLOWS,
```

Comment:
Isn't this redundant since you initialized `result.oAuthSettings.flows` with this value anyways?

Reply:
Cognito's oAuthSettings type uses read-only properties, so a new oAuthSettings object is being assigned here. Even if it was not read-only, copy assignment would be required for the other remaining propertyes (which don't show up in the PR):
```
result.oAuthSettings = {
  callbackUrls: external.callbackUrls,
  logoutUrls: external.logoutUrls,
  scopes: external.scopes,
```
I don't think the trade is worth it here